### PR TITLE
[gardening] Remove unused code. Fix file header formatting. Fix PEP 8 regression. Fix recently introduced typo: "accomodate" → "accommodate".

### PR DESCRIPTION
**Pull request by practicalswift**
#### What's in this pull request?
- Remove unused code.
- Fix file header formatting.
- Fix PEP 8 regression.
- Fix recently introduced typo: "accomodate" → "accommodate".
#### Resolved bug number: –

---

Before merging this pull request to apple/swift repository:
- [ ] Test pull request on Swift continuous integration.

<details>
  <summary>

Triggering Swift CI</summary>

The swift-ci is triggered by writing a comment on this PR addressed to the GitHub user @swift-ci. Different tests will run depending on the specific comment that you use. The currently available comments are:

**Smoke Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please smoke test |
| OS X platform | @swift-ci Please smoke test OS X platform |
| Linux platform | @swift-ci Please smoke test Linux platform |

 **Validation Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please test |
| OS X platform | @swift-ci Please test OS X platform |
| Linux platform | @swift-ci Please test Linux platform |

Note: Only members of the Apple organization can trigger swift-ci.
</details>

### CODE REVIEW

**Comment by practicalswift** [comment]
@modocache Do you have time to review the change made to `CMakeLists.txt` (removal of unused variable `SWIFT_SDK_ANDROID_PATH`)? :-)

File: CMakeLists.txt [hunk 1]
```diff
@@ -458,7 +458,6 @@ if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
     is_sdk_requested(ANDROID swift_build_android)
     if(swift_build_android AND ${swift_can_crosscompile_stdlib})
       configure_sdk_unix(ANDROID "Android" "android" "android" "armv7" "armv7-none-linux-androideabi")
-      set(SWIFT_SDK_ANDROID_PATH "${SWIFT_ANDROID_SDK_PATH}")
```
Comment (modocache) [inline CMakeLists.txt]: Whoops! 😅  Thanks as always, @practicalswift. The Android-related Python changes LGTM, too.

**Comment by practicalswift** [comment]
@modocache Thanks for reviewing! :-)

**Comment by modocache** [comment]
Ack, my bad, @practicalswift! I was mistaken, the CMake variable *is* necessary. See the commit message in my revert, https://github.com/apple/swift/commit/b8f7254151d4620c487456e8693ef1e2211c157a.

**Pull request merged**